### PR TITLE
release: v2.6.0

### PR DESCRIPTION
**Pull request by onerandomusername**
## Checklist

- [ ] If code changes were made, then they have been tested
    - [ ] I have updated the documentation to reflect the changes
    - [ ] I have formatted the code properly by running `task lint`
    - [ ] I have type-checked the code by running `task pyright`
- [ ] This PR fixes an issue
- [ ] This PR adds something new (e.g. new method or parameters)
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: docs/whats_new.rst [hunk 1]
```diff
@@ -17,6 +17,250 @@ in specific versions.
 
 .. towncrier release notes start
 
+.. _vp2p6p0:
+
+v2.6.0
+------
+
+This release contains support for forum channels (and their tags) as well as full support for the automoderation API and gateway events.
```
Comment (shiftinv) [inline docs/whats_new.rst:25]:
```suggestion
This release adds support for new forum channels (like tags) as well as auto moderation, among other things. See below for more.
Also note the breaking changes listed below, which may require additional code changes.
```

File: docs/whats_new.rst [hunk 2]
```diff
@@ -17,6 +17,250 @@ in specific versions.
...
+- Fixes message initialization failing with threads and no intents by explicitly checking we have a guild object where one is required. (:issue:`699`, :issue:`712`)
+- Fixed an issue where it would be possible to remove other features when enabling or disabling the ``COMMUNITY`` feature for a :class:`.Guild`. (:issue:`705`)
+- Fix invalid widget fields. (:issue:`736`)
+    - :attr:`Widget.invite_url` and :attr:`Widget.fetch_invite` are now optional.
+    - :attr:`WidgetMember.avatar` and :attr:`WidgetMember.activity` now work properly and no longer always raise an exception or return ``None``.
+- No longer use deprecated `@!` syntax for mentioning users. (:issue:`743`)
+- Fix creation of forum threads without :class:`Permissions.manage_threads`. (:issue:`746`)
+- Don't count initial message in forum threads towards :attr:`Thread.message_count` and :attr:`Thread.total_message_sent`. (:issue:`747`)
+- Fix edge case with parsing command annotations that contain a union of non-type objects, like ``Optional[Literal[1, 2, 3]]``. (:issue:`770`)
```
Comment (shiftinv) [inline docs/whats_new.rst:192]:
```suggestion
- |commands| Fix edge case with parsing command annotations that contain a union of non-type objects, like ``Optional[Literal[1, 2, 3]]``. (:issue:`770`)
```
small nit - noticed this earlier, might as well fix it now

**Review by shiftinv — CHANGES_REQUESTED** [review]

File: docs/whats_new.rst (hunk 2 above)
Comment (onerandomusername) [reply docs/whats_new.rst]: https://github.com/DisnakeDev/disnake/pull/775/commits/27c168d36dfebebe4921405bfe456a8f6e455c1c

File: docs/whats_new.rst (hunk 1 above)
Comment (onerandomusername) [reply docs/whats_new.rst]: https://github.com/DisnakeDev/disnake/pull/775/commits/b350ba8b958d5a0029b86739650dc85b1ad4025a

**Review by shiftinv — APPROVED** [review]

**Pull request merged**